docs/Deploying/Configuration.md

```diff
@@ -2065,16 +2068,35 @@ URL to fetch the CPE dictionary database from. Some users want to control where
   	cpe_database_url: ""
   ```
 
+##### cpe_translations_url
+
+URL to fetch the CPE translations from.
+Translations are used when matching software to CPE entries in the CPE database that would otherwise be missed for various reasons.
+When Fleet sees this value defined, it downloads the file directly.
+It expects a file in the same format as can be found in https://github.com/fleetdm/nvd/releases.
+If this value is not defined, Fleet checks for the latest release in Github and only downloads it if needed.
```

Comment:
Did you mean https://github.com/fleetdm/nvd/blob/master/cpe_translations.json? (does it check `/releases`?)

Reply:
There was[ a thread](https://github.com/fleetdm/fleet/pull/6985#discussion_r933658334) about moving those artifacts (both the `cpe_translations.json` and the `oval_sources.json` files) to `/releases` - but I think that's outside the scope of this PR and maybe we should do it in a separate issue.

Reply:
Ah NVM my prev. comment - seems like this will be included in this PR